db/migrate/20240507004300_add_user_id_to_medical_records.rb

```diff
@@ -1,9 +1,9 @@
 class AddUserIdToMedicalRecords < ActiveRecord::Migration[7.1]
   def change
-    user = Usuario.first || Usuario.create(email: "temporary_email@example.com", password: "atemporal", password_confirmation: "atemporal")
-    add_reference :medical_records, :usuario, null: true, foreign_key: true, default: user.id
+    user = User.first || User.create(email: "temporary_email@example.com", password: "atemporal", password_confirmation: "atemporal")
+    add_reference :medical_records, :user, null: true, foreign_key: true, default: user.id
 
-    change_column_default :medical_records, :usuario_id, nil
+    change_column_default :medical_records, :user_id, nil
```

Comment:
Same here.

Reply:
Ok, I understand but only the previous migrations, right? I mean only the content of the files?

Reply:
You may have a problem running this migration anyway because if you look for the first "Usuario", the migration won't find this model even when the table `usuarios` is there, this is because your code now have the model `User`.
On the other hand, if you change your query to look for the first `User`, the migration won't find a table `users` that matches the model, this is because at this point the `users` table doesn't exists.
So, given this case scenario, you need to assign the ID number directly rather than look for the record and hope it works whenever you run it.